iotedgedev/modules.py

```diff
@@ -121,7 +129,7 @@ def build(self):
                         self.output.info(
                             "PUSHING DOCKER IMAGE TO: " + image_destination_name)
 
-                        for line in self.dock.docker_client.images.push(repository=image_destination_name, tag=tag_name, stream=True, auth_config={
+                        for line in self.dock.docker_client.images.push(repository=image_destination_name, stream=True, auth_config={
```

Comment:
Did you find that tag_name is not required?

Reply:
I do not believe it is

Reply:
Why did you remove it?